scripts/appleseedMaya/renderGlobals.py

```diff
@@ -427,12 +432,14 @@ def create(self):
                                 enumeratedItem=self._getAttributeMenuItems("pixelFilter")),
                             attrName="pixelFilter")
                         self._addControl(
-                            ui=pm.floatFieldGrp(
-                                label="Pixel Filter Size", numberOfFields=1),
+                            ui=pm.floatSliderGrp(
+                                label="Pixel Filter Size", field=True, value=1.5, sliderStep=0.5, precision=1, cw=(3,160), minValue=0.5, 
+                                fieldMinValue=0.5, maxValue=6.0, fieldMaxValue=20.0),
                             attrName="pixelFilterSize")
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Tile Size", numberOfFields=1),
+                            ui=pm.intSliderGrp(
+                                label="Tile Size", field=True, value=64, cw=(3,160), minValue=8, 
```

Comment:
Hard min tile size should be 1. Hard max tile size should be very large (65536?). It's limited to about 2 billions (2^31-1) in appleseed.studio.

Reply:
Ok, I will increase max limit.